lib/management/configuration.js

```diff
@@ -179,6 +179,8 @@ function patchConfiguration(newConf, log, cb) {
                         l.locationType });
                     return;
                 }
+                location.sizeLimitGB = l.sizeLimitGB || false;
```

Comment:
I suggest leaving as `undefined` instead of `false` to avoid mixing data types (considered bad practice).

Reply:
will fix